client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -42,6 +44,50 @@ const WPOrderReviewListItem = styled.li`
 	list-style: none;
 `;
 
+const CostOverridesListStyle = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	font-size: 14px;
+	font-weight: 400;
+
+	& .cost-overrides-list-item {
+		display: flex;
+		justify-content: space-between;
+		padding: 2px 0px;
+
+		&__reason {
+			color: #008a20;
+		}
+	}
+`;
+
+function CostOverridesList( {
+	costOverridesList,
+}: {
+	costOverridesList: Array< {
+		human_readable_reason: string;
+		discount_amount: number;
+	} >;
+} ) {
+	return (
+		<>
+			{ costOverridesList.map( ( costOverride ) => {
+				return (
+					<div className="cost-overrides-list-item">
+						<span className="cost-overrides-list-item__reason">
+							{ costOverride.human_readable_reason }
+						</span>
+						<span className="cost-overrides-list-item__discount">
+							{ formatCurrency( -costOverride.discount_amount, 'USD' ) }
```

Comment:
This should use the currency of the cart, which we have in `responseCart.currency` and can pass into this component. Alternatively, you could add `currency` to each element of `costOverridesList` when you're generating it.